src/images/logo.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 273.65 217.03"><defs><style>.cls-1{fill:#ef5790;}.cls-2{fill:#231f20;}.cls-3{fill:#fff;}.cls-4{fill:none;stroke:#231f20;stroke-miterlimit:10;stroke-width:6px;}</style></defs><title>Asset 4</title><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><path class="cls-1" d="M136,217a9.29,9.29,0,0,1-.68-.94c-3.53-6.58-8.43-12-13.83-17.1-10.61-10-22.21-18.71-33.76-27.54C78,164,68.32,156.4,58.85,148.56a160.86,160.86,0,0,1-22-21.57C28,116.15,21.59,104.06,18.52,90.31a98,98,0,0,1-2.34-23.43c.1-4.82.27-9.66.74-14.46.55-5.69,2.75-10.89,5.35-15.93,5.17-10.05,12-18.69,21.42-25.11C50.32,6.83,57.38,3.05,65.32,1.3,71.26,0,77.33,0,83.38.08a52.89,52.89,0,0,1,13,1.31A32.69,32.69,0,0,1,101,3a65.77,65.77,0,0,1,29.9,25.08,37.69,37.69,0,0,1,4.85,12.48c.1.43.22.85.34,1.27,0,.06.08.1.22.26.18-.39.34-.73.51-1.07,2.3-4.62,4.26-9.44,7-13.81,8.25-13.34,20.28-21.71,35.47-25.37a63.71,63.71,0,0,1,62.57,19.36A55.83,55.83,0,0,1,256.11,51a125.52,125.52,0,0,1,.73,31.22c-1.39,14-6.74,26.39-14.66,37.82-6.44,9.28-14.34,17.22-22.77,24.68-9.83,8.7-20.44,16.41-30.93,24.27a406,406,0,0,0-47.1,41.38,23.23,23.23,0,0,0-4.8,6.11A3.19,3.19,0,0,1,136,217Z"/><path class="cls-2" d="M151.88,48.11a44.44,44.44,0,0,1-15.06,1.51,44.44,44.44,0,0,1-15.06-1.51C78.52,26.17,0,39.94,0,39.94V55.42C7.1,55.64,9,64,9,64c5.59,74,51.85,63.25,51.85,63.25,58.52,2.58,66.26-57.22,66.26-57.22.65-6,9.68-7.1,9.68-7.1s9,1.08,9.68,7.1c0,0,7.74,59.81,66.26,57.22,0,0,46.25,10.76,51.85-63.25,0,0,1.94-8.39,9-8.61V39.94S195.12,26.17,151.88,48.11ZM62.6,121s-42.6,6-44.53-52.8c0,0-4.52-26.37,48-25.51,0,0,50.77-5.38,49.48,28.18C115.53,70.91,119.4,117.6,62.6,121Zm193-52.8C253.64,127.09,211,121,211,121c-56.79-3.44-52.92-50.13-52.92-50.13-1.29-33.56,49.48-28.18,49.48-28.18C260.09,41.87,255.58,68.24,255.58,68.24Z"/><path class="cls-2" d="M135.25,162.55c-17.91,0-22.28-12.83-22.46-13.37l5.71-1.85-2.85.93,2.84-1c.13.38,3.35,9.25,16.76,9.25,14.15,0,20.18-8.95,20.24-9l5,3.29C160.19,151.28,152.61,162.55,135.25,162.55Z"/><circle class="cls-3" cx="206.2" cy="78.57" r="28.14"/><circle class="cls-2" cx="206.2" cy="75.34" r="24.91"/><circle class="cls-3" cx="67.2" cy="78.57" r="28.14"/><circle class="cls-2" cx="67.2" cy="74.46" r="24.04"/><path class="cls-4" d="M256.5,40.42"/><path class="cls-2" d="M46.31,33l-2.38-3.21c.94-.7,23.21-16.91,39.22-3.8l-2.53,3.09C67,18,46.51,32.89,46.31,33Z"/><path class="cls-2" d="M227.54,33c-.21-.15-20.71-15.05-34.3-3.92L190.7,26c16-13.11,38.28,3.1,39.22,3.8Z"/></g></g></svg>
```

Comment:
suggest the following structure:

src/
-assets/
--images/
--whatever-other-typeof-assets/

because assets can then include other files other then just images. And you dont want to put something like videos inside of the images folder